apm-agent-plugins/apm-jms-plugin/src/main/java/co/elastic/apm/agent/jms/JmsMessageConsumerInstrumentation.java

```diff
@@ -216,6 +221,9 @@ public static void afterReceive(@Advice.Origin Class<?> clazz,
                             if (message != null && helper != null && destinationName != null) {
                                 abstractSpan.appendToName(" from ");
                                 helper.addDestinationDetails(message, destination, destinationName, abstractSpan);
+                                if (messageAge > 0) {
```

Comment:
shouldn't it be `>= 0` ?

Reply:
In fact it is more complicated, I just updated this field's definition and didn't update this yet- it may be negative and we want to use it...
I will fix. Thanks for noticing!